python-lib/utils/language_support.py

```diff
@@ -128,3 +128,136 @@
 """list: Languages that have available SpaCy rule tables for lemmatization
 The rule tables are available at https://github.com/explosion/spacy-lookups-data/tree/master/spacy_lookups_data/data
 """
+
+PUNCTUATION_CHARACTERS = [
```

Comment:
Does that mean you are enforcing the same punctuation characters for all languages ? Was it the same before this PR ?

Reply:
Also, can you import the tables from spacy, instead of hardcoding it here?

Reply:
> Does that mean you are enforcing the same punctuation characters for all languages ? Was it the same before this PR ?
It was, as we discussed earlier

> Also, can you import the tables from spacy, instead of hardcoding it here?

Sure, I did it in this commit d8eb136

Reply:
Much better! But what about "/r" and "/t"?

Reply:
Actually, I don't think we should break lines on \t ? 

And for  \r, it is replaced by \n in `"\n".join(line.strip() for line in filter(None, text.splitlines()))` (text.splitlines() split on \r and \n, and then the sentences are recomposed with  "\n" only)

Reply:
Good point, you can close this